lib/rubocop/runner.rb

```diff
@@ -27,7 +27,11 @@ def initialize(options, config_store)
 
     def run(paths)
       target_files = find_target_files(paths)
-      inspect_files(target_files)
+      if @options[:list_target_files]
+        puts target_files.map { |file| PathUtil.relative_path(file) }
```

Comment:
I kinda like how `run` finds files and then asks someone else to either inspect them or print their names.

Let's hear what @bbatsov says.